miner/src/block_assembler.rs

```diff
@@ -413,6 +418,192 @@ impl BlockAssembler {
         Ok(template)
     }
 
+    fn package_txs(
+        &self,
+        chain_state: &ChainState,
+        size_limit: usize,
+        cycles_limit: Cycle,
+    ) -> (Vec<TxEntry>, usize, Cycle) {
+        /// update descendants txs into modified entries
+        fn update_modified_entries(
+            tx_pool: &TxPool,
+            in_block_txs: &HashSet<ProposalShortId>,
```

Comment:
What is `in_block_txs` and what is `package_txs`. From this local function itself, I have no ideas. I have to search where this function is used.

Reply:
`in_block_txs` is a bad name, the transactions selected are not in the block yet.

Suggest names:

- `in_block_txs`: `fetched_txs` transactions that already fetched
- `package_txs`: `new_fetched_txs` transactions that has just been fetched